src/postamble.js

```diff
@@ -316,6 +316,10 @@ Module['exit'] = Module.exit = exit;
 var abortDecorators = [];
 
 function abort(what) {
+  if (Module['onRuntimeInitializeFailed']) {
+    setTimeout(function () { Module['onRuntimeInitializeFailed'](what); }, 0);
```

Comment:
It's probably not really needed; I can either remove it or add a comment depending on your preference. The reason I put it in a `setTimeout` was to avoid having an exception in the callback interfere with the rest of `abort` or anything else that may come afterwards.